### ISSUE TITLE
`--daemon` mode ignores `-e` for first window with `msg create-window`

### ISSUE THREAD

**Issue by dorsiflexion** [alacritty/alacritty#8371]
I use another terminal to not interfere with alacritty instances. Then do:
- `alacritty --daemon`
- `alacritty msg create-window -e nnn`

→ Alacritty opens and is a child process of the daemon, as it should. It is a blank, interactive terminal window though. `nnn` (or any other command) did not execute. When using `alacritty msg create-window -e nnn` a second time, even after closing the first window, it works as intended.

### System

OS: Linux
Version: alacritty 0.14.0 (22a44757)
Linux/BSD: X11, i3wm

### Logs

Sadly, nothing of value. `alacritty msg create-window -e nnn` does not produce any output.

`alacritty --daemon` produces the following. 6.[…] is the first, unsuccessful use of `alacritty msg create-window`, 9.[…] is the second one.
```
❯ alacritty --daemon
ALACRITTY_SOCKET=/run/user/1000/Alacritty-:0-4690.sock; export ALACRITTY_SOCKET
Created log file at "/tmp/Alacritty-4690.log"
[6.685035667s] [INFO ] [alacritty] Using GLX 1.4
[6.731810987s] [DEBUG] [alacritty] Picked GL Config:
                                     buffer_type: Some(Rgb { r_size: 8, g_size: 8, b_size: 8 })
                                     alpha_size: 8
                                     num_samples: 0
                                     hardware_accelerated: true
                                     supports_transparency: Some(true)
                                     config_api: Api(OPENGL | GLES1 | GLES2)
                                     srgb_capable: false
[6.733080791s] [INFO ] [alacritty] Window scale factor: 1.1354166666666667
[6.735497376s] [DEBUG] [alacritty] Loading "monospace" font
[6.747956223s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[6.751520900s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[6.755152785s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(NO_BITMAP), render_mode: "Normal", lcd_filter: 1 }
[6.758946778s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold, load_flags: LoadFlag(NO_BITMAP), render_mode: "Normal", lcd_filter: 1 }
[6.762142955s] [INFO ] [alacritty] Running on AMD Radeon RX 580 Series (radeonsi, polaris10, LLVM 18.1.8, DRM 3.59, 6.11.11-hardened1-1-hardened)
[6.762156901s] [INFO ] [alacritty] OpenGL version 4.6 (Core Profile) Mesa 24.3.1-arch1.3, shader_version 4.60
[6.762167441s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[6.776950928s] [DEBUG] [alacritty] Enabled debug logging for OpenGL
[6.777039627s] [DEBUG] [alacritty] Filling glyph cache with common glyphs
[6.798030936s] [INFO ] [alacritty] Cell size: 12 x 26
[6.798055242s] [INFO ] [alacritty] Padding: 4 x 1
[6.798064319s] [INFO ] [alacritty] Width: 800, Height: 600
[6.804284075s] [INFO ] [alacritty] PTY dimensions: 23 x 66
[6.811177551s] [DEBUG] [alacritty_terminal] New num_cols is 105 and num_lines is 26
[6.813260831s] [INFO ] [alacritty] Padding: 3 x 7
[6.813277253s] [INFO ] [alacritty] Width: 1266, Height: 690
[6.917197337s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[6.917597738s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[9.726376377s] [INFO ] [alacritty] Window scale factor: 1.1354166666666667
[9.728275928s] [DEBUG] [alacritty] Loading "monospace" font
[9.730893835s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[9.734716353s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[9.738351434s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(NO_BITMAP), render_mode: "Normal", lcd_filter: 1 }
[9.742213867s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Bold, load_flags: LoadFlag(NO_BITMAP), render_mode: "Normal", lcd_filter: 1 }
[9.744627586s] [INFO ] [alacritty] Running on AMD Radeon RX 580 Series (radeonsi, polaris10, LLVM 18.1.8, DRM 3.59, 6.11.11-hardened1-1-hardened)
[9.744640250s] [INFO ] [alacritty] OpenGL version 4.6 (Core Profile) Mesa 24.3.1-arch1.3, shader_version 4.60
[9.744648687s] [INFO ] [alacritty] Using OpenGL 3.3 renderer
[9.746431446s] [DEBUG] [alacritty] Enabled debug logging for OpenGL
[9.746509915s] [DEBUG] [alacritty] Filling glyph cache with common glyphs
[9.762323680s] [INFO ] [alacritty] Cell size: 12 x 26
[9.762351894s] [INFO ] [alacritty] Padding: 4 x 1
[9.762362313s] [INFO ] [alacritty] Width: 800, Height: 600
[9.765687004s] [INFO ] [alacritty] PTY dimensions: 23 x 66
[9.771741265s] [DEBUG] [alacritty_terminal] New num_cols is 105 and num_lines is 17
[9.774571766s] [INFO ] [alacritty] Padding: 3 x 6
[9.774589039s] [INFO ] [alacritty] Width: 1266, Height: 455
[9.775390092s] [DEBUG] [alacritty_terminal] New num_cols is 105 and num_lines is 17
[9.775689692s] [INFO ] [alacritty] Padding: 3 x 8
[9.775704119s] [INFO ] [alacritty] Width: 1266, Height: 459
[9.775863562s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
[9.785250999s] [DEBUG] [crossfont] Loaded Face Face { ft_face: Font Face: Regular, load_flags: LoadFlag(0x0), render_mode: "Normal", lcd_filter: 1 }
```

**Comment by chrisduerr** [comment]
Should be fixed with #8374.

**Comment by dorsiflexion** [comment]
Thanks a ton!
